main.go

```diff
@@ -145,9 +115,10 @@ func main() {
 		http.Handle(*metricPath, promhttp.Handler())
 	}
 
+	http.HandleFunc("/scrape", exp.ScrapeHandler)
```

Comment:
Isn't `/metrics` the standard endpoint? Also I think most exporters provide this path as an argument

Reply:
Good question! Yes, `/metrics` normally is the standard endpoint and that's what's still used for scraping the exporter metrics. It's configurable via `--web.telemetry-path` and hooked up in main.go line 115.
`/scrape` is what's used for scraping multiple redis instances,  [see the README here](https://github.com/oliver006/redis_exporter/blob/oh_1.0_here_we_come/README.md#prometheus-configuration-to-scrape-several-hosts).